### DISCUSSION TITLE
use `require` to import modules when input is not a module

### DISCUSSION THREAD

**Discussion by RiESAEX** [swc-project/swc#6760] [Ideas]
For polyfills and external helpers when input is not a module, swc will throw unreachable.
There is no standard way to import a module in script, but swc's module passing and webpack understand `require`.
I think we can use `require` to import modules when input is not a module
relative to https://github.com/swc-project/swc/issues/6500 https://github.com/swc-project/swc/issues/6500

**Comment by kdy1** [comment]
I agree, we can use `require` for scripts.
